experiments/GG/TestTextControl.cpp

```diff
@@ -0,0 +1,29 @@
+#include <GG/GUI.h>
+#include <GG/Clr.h>
+#include <GG/ClrConstants.h>
+#include <GG/dialogs/ThreeButtonDlg.h>
+#include <GG/StyleFactory.h>
+
+#include "TestApp.h"
+
+int main(int argc, char* argv[])
+{
+
+    TestApp app(argc, argv);
+
+    const std::string message = "Are <i>we</i> Готово yet?"; // That Russian word means "Done", ha.
```

Comment:
I think you might need some extra stuff before/after the "" to make this work properly as a UTF-8 string constant.